package.json

```diff
@@ -13,21 +13,19 @@
   },
   "homepage": "https://github.com/plotly/dash-html-components",
   "scripts": {
-    "clean-lib": "rm -rf lib/* && mkdir -p lib",
-    "clean-src": "rm -rf src/* && mkdir -p src/components",
-    "clean": "npm run clean-lib && npm run clean-src",
-    "copy-lib": "cp lib/* dash_html_components",
+    "clean": "rm -rf src/* && mkdir -p src/components",
     "generate-python-classes": "python -c 'import dash; dash.development.component_loader.generate_classes(\"dash_html_components\", \"dash_html_components/metadata.json\")'",
     "generate-components": "cd scripts && sh generate-all.sh && cd -",
-    "install-local": "npm run copy-lib && python setup.py install",
-    "prepublish": "npm run clean && npm run generate-components && npm test && npm run build:js && npm run build:js-dev && npm run build:py",
-    "publish-all": "npm publish && python setup.py sdist upload",
-    "start": "webpack-serve ./webpack.serve.config.js --open",
-    "test": "eslint --fix src",
+    "install-local": "python setup.py install",
     "uninstall-local": "pip uninstall dash-html-components -y",
+    "prepublish": "npm run clean && npm run generate-components && npm run build:all",
```

Comment:
We should remove prepublish entirely and use the new npm methods it warns about instead. This will throw when you install on windows.